terraform/variables.tf

```diff
@@ -5,3 +5,13 @@ variable "DEVSECOPS_OBJECT_ID" {
   description = "Object ID for the DevSecOps principal, which includes the Production service connection"
   type        = string
 }
+
+variable "ENGINEERING_GROUP_OBJECT_ID" {
+  description = "Object ID for the Cal-ITP engineering Active Directory Group"
+  type        = string
+}
+
+variable "MYSTERY_APP_SERVICE_OBJECT_ID" {
+  description = "Object ID for something with principal name 'Microsoft Azure App Service'"
+  type        = string
+}
```

Comment:
No idea what this is for, who/what created it (could have been me)… Will continue to investigate.

Reply:
It doesn't show up in the other App Service instances, so I think it's not necessary. Worst case, we will see the problem in `dev` and can revert.